src/main/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilder.java

```diff
@@ -51,12 +51,19 @@ public static String buildSearchConceptsQuery(
     var domainClause =
         createDomainClause(conceptTablePointer, conceptTableVariable, domainOption.getCategory());
 
+    List<FieldVariable> select = List.of(nameField, idField, personIdField);
```

Comment:
nit- maybe instead of personId field it could be called count field. What do you think?

Reply:
I like this! I think personIdField can be a little confusing and it'll be harder to attribute this to a count field.